core/ipc/sink/common/src/main/proto/sink-message.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+option java_multiple_files = true;
+option java_package = "org.opennms.core.ipc.sink.model";
+option java_outer_classname = "SinkMessageProtos";
+
+
+message SinkMessage {
+string message_id = 1;
+bytes  content = 2;
+string system_id = 3;
```

Comment:
Are we using `system_id` and `location`?

Reply:
No. I was using the same SinkMessage as gRPC but we don't need them as of now. Will remove them